Managed service provider deployment is failing in 2.0.11

Managed service provider deployment is failing with the error given below.
```
E                   AssertionError: OSD pod rook-ceph-osd-0-76f9f4d96f-bmz89 container osd doesn't have limits memory of 7Gi
```

--- a/ocs_ci/framework/conf/default_config.yaml
+++ b/ocs_ci/framework/conf/default_config.yaml
@@ -218,7 +218,7 @@ ENV_DATA:
   ms_env_type: 'staging'
   rosa_cli_version: '1.2.6'
   appliance_mode: true
-  ms_osd_pod_memory: "7Gi"
+  ms_osd_pod_memory: "5700Mi"
 
   # used in external mode deployment
   restricted-auth-permission: false

